src/WordPress.php

```diff
@@ -377,6 +370,29 @@ private function configure_site_files() {
 		}
 	}
 
+	/**
+	 * Generate and place docker-compose.yml file.
+	 *
+	 * @param array $additional_filters Filters to alter docker-compose file.
+	 */
+	private function dump_docker_compose_yml( $additional_filters = [] ) {
+
+		$site_docker_yml = $this->site_data['site_fs_path'] . '/docker-compose.yml';
+
+		$filter                 = [];
+		$filter[]               = $this->site_data['app_sub_type'];
+		$filter[]               = $this->cache_type ? 'redis' : 'none';
+		$filter[]               = $this->site_data['db_host'];
+		$site_docker            = new Site_WP_Docker();
+
+		foreach ( $additional_filters as $key => $addon_filter ) {
```

Comment:
You could have also added `nohttps` in filter and checked in generate_docker_compose_yml that if it exists then add `nohttps` block. Any specific reason to use this?

Reply:
Ignore this for now as we're going to convert it to key value pair tommorow